### ISSUE TITLE
Feature: (optionally?) allow geologist signs to not vanish

### ISSUE THREAD

**Issue by stepo2** [nicymike/serflings#95]
It would be nice to have an option (or even as a default behaviour) to not have these signs of geologist not vanish after some time. Since there's not even a fog of war you could simply pause the game and make notes on a paper where what kind of resource is located. So vanishing signs just impose an unneccessary obstacle for the player but add no diffiulty or tactical/strategical considerations to the actual gameplay.

**Comment by nicymike** [comment]
I guess this was implemented because of decay. Also the values will change if mines are gathering the resources.

**Comment by stepo2** [comment]
Yes, wooden signs would decay in reality. But the information about them would not decay. In a real world civilisation, the castle would hold papers in a library about such resource locations even if the signs are long decayed. So in this game the signs serve both purposes, the graphical element of the real sign post as well the written down "permanent" information that resides "in the castle".

Building a mine could delete all signs in the relevant radius around it (only the ones that the mine is gathering). But it's not neccessary that signs decay by itself for the above reasons.

Another possibility would be that clicking on a mountain node shows some indicator on the screen. Like a question mark if no geologist was there or the coal/gold/iron icon.

I'm not sure what the best solution is here in teh end, but the information itself about what ore was found at a specific mountain node should not just vanish.

**Comment by nicymike** [comment]
Nice ideas. I put it on my list and think about it.

**Comment by ShpakMA** [comment]
This function should to be switchable (off/on/time). But using this is bad game practice. I just put mines immediately after resources are discover, but not connecting new mines until I need it. Using special click you may obtain information about planned building. You just place correct mine in correct place, and no paper need.

**Comment by stepo2** [comment]
But placing mines can change how some of the AI players act, afaik.

**Comment by ShpakMA** [comment]
> But placing mines can change how some of the AI players act, afaik.

AFAIK, AI react only for mines with builders and/or miners…

Can anybody test this on original game?
